CMakeLists.txt

```diff
@@ -59,4 +59,14 @@ set_target_properties(mycpplib PROPERTIES
 target_link_libraries(mycpplib
   ${catkin_LIBRARIES}
   ${Boost_LIBRARIES}
+
+)
+
+install(TARGETS mycpplib
+  ARCHIVE DESTINATION ${CATKIN_PACKAGE_PYTHON_DESTINATION}
+  LIBRARY DESTINATION ${CATKIN_PACKAGE_PYTHON_DESTINATION}
 )
+
+file(GLOB py_files "scripts/*.py")
```

Comment:
To keep it minimalist, I would prefer to not use the `file` command but just use the explicit filename.